apps/fabric-website/src/components/Nav/Nav.tsx

```diff
@@ -24,9 +25,11 @@ export class Nav extends React.Component<INavProps, INavState> {
       : null;
 
     return (
-      <nav className={ styles.nav } role='navigation'>
-        { links }
-      </nav>
+      <FocusZone ref='focusZone' direction={ FocusZoneDirection.vertical } role='menubar' >
```

Comment:
Why are we adding `role='menubar'`? a menubar requires nested menuitems as per https://www.w3.org/TR/wai-aria-practices-1.1/#menu . Also we should avoid using string refs https://facebook.github.io/react/docs/refs-and-the-dom.html#legacy-api-string-refs